### DISCUSSION TITLE
Loopback

### DISCUSSION THREAD

**Discussion by iulko** [celzero/rethink-app#1287] [General]
Hello,

When loopback option (to get Rethink to use Rethink to move traffic is getting enabled, it says that "Use all available networks" needs to be enabled. Its super cool you finally got that all working,

One question.

Does  "Use all available networks" mean that when i have wifi+LTE enabled at the same time, some apps can choose LTE and some Wifi

**Comment by ignoramous** [comment]
> Does "Use all available networks" mean that when i have wifi+LTE enabled at the same time, some apps can choose LTE and some Wifi

Not apps (unless `Enable network visibility` is turned ON, apps can't bind to other interfaces/networks other than Rethink's own VPN interface), but Rethink is free to choose either Wifi or LTE, yes.

>  Its super cool you finally got that all working,

Took us about 3 odd months to fix all its quirks. It was super annoying especially for a feature that *no* one should really have a need for...
